crates/metadata-server/src/raft/server.rs

```diff
@@ -174,24 +173,32 @@ impl RaftMetadataServer {
         let mut storage =
             RocksDbStorage::create(metadata_store_options.live_load(), rocksdb_options).await?;
 
-        // todo unify with LogServer logic; turn into timestamp?
         // make sure that the storage is initialized with a storage id to be able to detect disk losses
-        let storage_id = if let Some(storage_id) = storage
-            .get_storage_id()
+        if let Some(storage_marker) = storage
+            .get_marker()
             .map_err(|err| BuildError::InitStorage(err.to_string()))?
         {
-            storage_id
+            if storage_marker.id() != Configuration::pinned().common.node_name() {
```

Comment:
Q: why the marker stores the node name, and not the node id.

Reply:
Because when we initialize the storage, then we might not have obtained the node id yet.